src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.parser.ParserImpl;
+import core.basesyntax.service.reader.CsvFileReader;
+import core.basesyntax.service.reader.FileReader;
+import core.basesyntax.service.reporter.CsvReporter;
+import core.basesyntax.service.reporter.Reporter;
+import core.basesyntax.service.strategy.OperationHandler;
+import core.basesyntax.service.strategy.OperationStrategy;
+import core.basesyntax.service.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.service.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.service.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.service.strategy.impl.SupplyOperationHandler;
+import core.basesyntax.service.writter.CsvFileWritter;
+import core.basesyntax.service.writter.FileWritter;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String PATH_FROM = "src/main/resources/input.csv";
+    private static final String PATH_TO = "src/main/resources/output.csv";
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private static final Map<FruitTransaction.Operation, OperationHandler> handlers = Map.of(
```

Comment:
Just making a suggestion, if this variables have `private static final` modifiers, can we interpret as constants?